string camera_name
int64 mac_address
---
bool success
string message
